examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx

```diff
@@ -100,9 +101,15 @@ const RideButtons = ({
     setTempSelectedDate(firstDate());
   }, [minMinutesBeforeFutureRide]);
 
+  const checkFutureBookingDays = async () => {
```

Comment:
can it be in a context? its duplicated, same for the maxDaysFutureRide, setMaxDaysFutureRide